model/schema/resourceDescriptor.yaml

```diff
@@ -0,0 +1,84 @@
+id: https://github.com/alliance-genome/agr_curation_schema/resourceDescriptor.yaml
+name: Alliance-Schema-Prototype-Resource-Descriptor
+title: Alliance Schema Prototype Resource Descriptor
+
+imports:
+  - core
+  - linkml:types
+
```

Comment:
Probably need affectedGenomicModel added here as that is where the 'name' slot is defined

Reply:
done